### ISSUE TITLE
Connect FIlter Menu Boroughs dropdown to API

### ISSUE THREAD

**Issue by TylerMatteo** [NYCPlanning/ae-cp-map#9]
## Acceptance Criteria
- [ ] Populate options in Borough dropdown in FilterMenu component ([Figma](https://www.figma.com/design/LYHHoPop9l0jpEivk5CFzJ/Capital-Projects-Portal?node-id=253-4689&t=i9OdRPOzwaKcOnqa-0)) using data from `findBoroughs` endpoint.

**Comment by TangoYankee** [comment]
Blocked by: #18

**Comment by TylerMatteo** [comment]
Closing in favor of #28
